5/README.md

```diff
@@ -22,46 +28,76 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Tango Property Specification
 
- A Property is designed to represent any information of configuration in Tango.
+This specification is intended to formally document the Tango Property model.
+Runtime representation of Property in conforming Tango implementations MUST follow this specification.
 
 ### Goals
 
- The specification of Property aims to define a standard structure and type to use in any implementation of Tango.
+A Property represents a configuration parameter in Tango.
 
-Additionally, it aims to:
-
-* Provide the initial information when one starts a Tango Device
-
-* Be usable as keeping global variable in the Tango control system
-
-* Provide meta data complementing any data in Tango
+There are 4 kinds of Tango Properties:
 
+ *  Class Property: a Property related to a Device Class (cf RFC-2).
```

Comment:
For consistency with the following... s/(cf RFC-2)/(RFC-2)/

Reply:
Done in https://github.com/tango-controls/rfc/pull/59/commits/898755d7f07eb6dfdf5bcb2fe56fbb2de7d394c4